src/main/java/mServer/crawler/sender/phoenix/parser/ZdfDownloadDtoDeserializer.java

```diff
@@ -100,12 +100,13 @@ private void parseGeoLocation(final DownloadDto dto, final JsonObject rootNode)
       final JsonElement geoLocation = attributes.getAsJsonObject().get(JSON_ELEMENT_GEOLOCATION);
       if (geoLocation != null) {
         final JsonElement geoValue = geoLocation.getAsJsonObject().get(JSON_PROPERTY_VALUE);
-        if (geoValue != null && !geoValue.getAsString().equals("none")) {
-          final Optional<GeoLocations> foundGeoLocation = Optional.of(GeoLocations.valueOf(geoValue.getAsString()));
-          if (foundGeoLocation.isPresent()) {
-            dto.setGeoLocation(foundGeoLocation.get());
-          } else {
-            LOG.debug(String.format("Can't find a GeoLocation for \"%s", geoValue.getAsString()));
+        if (geoValue != null && !geoValue.getAsString().toUpperCase().equals("NONE")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)